packages/dd-trace/src/exporters/log/index.js

```diff
@@ -47,7 +42,7 @@ class LogExporter {
       }
     }
     logLine += ']}\n'
-    this._outputStream.write(logLine)
+    console.log(logLine) // eslint-disable-line no-console
```

Comment:
console.log is monkey-patched in lambda to add additional logging context to the start of the line, so it produces different output to process.stdout.write.